5calls/app/src/main/res/layout/activity_about.xml

```diff
@@ -1,177 +1,76 @@
-            android:text="@string/contact_us_prompt"
-            android:textColor="@color/colorPrimary"
-            android:textSize="@dimen/text_size_paragraph"
-            android:textStyle="bold|italic"
-        />
+            style="@style/aboutParagraphTextStyle"
+            android:text="@string/contact_us_prompt" />
 
         <Button
             android:id="@+id/contact_us_btn"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_gravity="start"
-            android:layout_margin="0dp"
-            android:background="@android:color/transparent"
-            android:minHeight="@dimen/accessibility_min_size"
-            android:text="@string/contact_us_btn"
-            android:textAlignment="textStart"
-            android:textColor="@color/colorAccent"
-        />
+            style="@style/transparentBtnStyle"
+            android:text="@string/contact_us_btn" />
 
         <TextView
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:text="@string/rate_us_prompt"
-            android:textColor="@color/colorPrimary"
-            android:textSize="@dimen/text_size_paragraph"
-            android:textStyle="bold|italic"
-            />
-
+            style="@style/aboutParagraphTextStyle"
+            android:text="@string/rate_us_prompt" />
+        <!-- wasn't in next btn <item name="android:layout_margin">@dimen/layout_margin_button</item>-->
         <Button
             android:id="@+id/rate_us_btn"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_gravity="start"
-            android:background="@android:color/transparent"
-            android:minHeight="@dimen/accessibility_min_size"
-            android:text="@string/rate_us_btn"
-            android:textAlignment="textStart"
-            android:textColor="@color/colorAccent"
-            />
+            style="@style/transparentBtnStyle"
+            android:text="@string/rate_us_btn" />
 
         <TextView
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="32dp"
-            android:linksClickable="true"
-            android:text="@string/open_source_info"
-            android:textColor="@color/colorPrimary"
-            android:textSize="@dimen/text_size_paragraph"
-            />
+            style="@style/linksParagraphTextStyle"
+            android:text="@string/open_source_info" />
 
         <TextView
             android:id="@+id/version_info"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="16dp"
-            android:textColor="@color/colorPrimary"
-            android:textSize="@dimen/text_size_paragraph"
-            />
+            style="@style/paragraphTextStyle" />
```

Comment:
I think in this case you've lost the marginTop. Maybe you can add it as a marginBottom on the previous view instead?

Reply:
I modified the style, so the marginTop is there again.